stable_baselines3/common/utils.py

```diff
@@ -284,3 +284,17 @@ def safe_mean(arr: Union[np.ndarray, list, deque]) -> np.ndarray:
     :return:
     """
     return np.nan if len(arr) == 0 else np.mean(arr)
+
+
+def polyak_update(params: Iterable[th.nn.Parameter], target_params: Iterable[th.nn.Parameter], tau: float) -> None:
+    """
+    Perform a polyak average update on target_params using params
```

Comment:
```suggestion
    Perform a Polyak average update on ``target_params`` using ``params``
```

+ maybe add a comment of why we are doing it that way?
+ maybe a link to the issue?

Reply:
Will add both!